setuptools/dist.py

```diff
@@ -406,7 +411,9 @@ class Distribution(_Distribution):
         'long_description_content_type': None,
         'project_urls': dict,
         'provides_extras': ordered_set.OrderedSet,
-        'license_files': ordered_set.OrderedSet,
+        'license_file': None,
+        'license_files': None,
+        'license_files_computed': list,
```

Comment:
Could it be possible to avoid this diff by using `license_files` where `license_files_computed` is found (and thus not introduce a new name)?